### PR TITLE
[5.0] schemaorg utf8 encoding consistent

### PR DESCRIPTION
**Pull request by heelc29**
### Summary of Changes

Apply changes from #39951 for schema.org plugin

### Testing Instructions

code review

### CODE REVIEW

**Comment by HLeithner** [comment]
thanks

File: plugins/schemaorg/organization/organization.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	<authorUrl>www.joomla.org</authorUrl>
 	<version>5.0.0</version>
 	<description>PLG_SCHEMAORG_ORGANIZATION_XML_DESCRIPTION</description>
- 	<namespace path="src">Joomla\Plugin\Schemaorg\Organization</namespace>
+ -	<namespace path="src">Joomla\Plugin\Schemaorg\Organization</namespace>
```
Comment (richard67) [inline plugins/schemaorg/organization/organization.xml:12]: The ` -` is wrong here. PR with fix see #41257 .

Comment (heelc29) [reply plugins/schemaorg/organization/organization.xml:12]: Shit

Comment (richard67) [reply plugins/schemaorg/organization/organization.xml:12]: Can happen. It was the only one, I've checked with a regexp search,

Comment (heelc29) [reply plugins/schemaorg/organization/organization.xml:12]: Thanks

Comment (HLeithner) [reply plugins/schemaorg/organization/organization.xml:12]: can you please create a new pr to fix this issue?

**Pull request merged**